translations/en-us.yml

```diff
@@ -1393,7 +1393,14 @@ registries:
     form_view:
         none_selected: 'None selected'
     overview:
+        initial:
+            text: 'Pending'
+            short_description: 'Pending admin contributor approval'
```

Comment:
Brian, I believe Admin should be either capitalized or spelled out fully as 'administrator'. This shorthand is slightly informal.